### ISSUE TITLE
Rename "Easy To Fix" label to "good first issue"

### ISSUE THREAD

**Issue by ylemkimon** [sympy/sympy#13465]
GitHub has added "help wanted" and "good first issue" labels to their [default labels](https://help.github.com/articles/about-labels/), and states:
> Now, GitHub will help potential first-time contributors discover issues labeled with `help wanted` or `good first issue`

Also, excerpted from https://github.com/servo/servo/issues/18905:
> The keynotes at GitHub Universe indicated that they intend to do a lot of future work using these default labels to surface appropriate issue for new contributors. It is probably a good idea for us to get in the habit of using these standard labels, to take full advantage of the future tools that might become available and the training that prospective contributors get when working with other repositories.

It currently suggests it at the top of issues page for first-time contributors like:

![issue](https://user-images.githubusercontent.com/888148/31647720-c1b63292-b344-11e7-93dc-97253506ce44.png)

As we are already using "Easy To Fix" label to flag issues appropriate for new contributors, I think we should rename "Easy To Fix" to "good first issue" or add "good first issue" to those with "Easy To Fix".

**Comment by gxyd** [comment]
BTW, can any contributor add 'help wanted' label to his PR?

**Comment by asmeurer** [comment]
It would be great if GitHub could support making it so that we don't have to rename the labels. We have a lot of developer documentation and mailing list posts and so on that say "easy to fix". If we change the label name the links will break and it will be confusing to new users.

**Comment by ylemkimon** [comment]
@gxyd No, still those with write access can apply labels. From documentation:
>To create, edit, apply, or delete a label, you must have write access to the repository.

> `help wanted`: Indicates that a **maintainer** wants help on an issue or pull request

@asmeurer Maybe we can create an issue with labels "Easy to Fix" and "good first issue" and redirect to one label, or use both labels simultaneously.

**Comment by asmeurer** [comment]
@ylemkimon a redirect issue sounds like a simple solution until they make it configurable.

**Comment by ylemkimon** [comment]
@asmeurer Should we move all “Easy to Fix” issues to “good first issue” or leave as is?

**Comment by asmeurer** [comment]
I would leave it as-is for now. Let's open a meta-issue with "good first issue" that points to "Easy to Fix" for now. Too much of our new user documentation references "Easy to Fix".

**Comment by ylemkimon** [comment]
I've created #13520. Any suggestions on the title and contents are appreciated.

**Comment by oscarbenjamin** [comment]
This came up recently on the [mailing list](https://groups.google.com/forum/#!topic/sympy/x80-mTsXQx8).

I think it would be reasonable to change from Easy-to-fix to Good-first-issue except that it's difficult to get everyone to understand a change when many people are involved. The question that needs answering is: how important is this?

**Comment by himanshugupta11002** [comment]
@oscarbenjamin , The Good-first-issues highlights opportunities for people who are new to open source and contribute to your projects.

**Comment by Mokshith1708** [comment]
I am new to open source and would love to start with this issue if it is open. I think i will be able to do it with a little bit of guidance. Can I take up this issue?

**Comment by newfadel** [comment]
You could use the GitHub API to programmatically manage labels if needed, although for just one label, doing it via the UI might be simpler.

For example, using Python with a library like `PyGithub` to rename a label:

```python
from github import Github

# Replace with your token and repo details
g = Github("YOUR_GITHUB_TOKEN")
repo = g.get_user("sympy").get_repo("sympy")

# Get the existing label
try:
    label = repo.get_label("Easy to Fix")
    # Rename the label
    label.edit("good first issue", "ededed") # Use the existing color or choose a new one
    print("Label 'Easy to Fix' renamed to 'good first issue'.")
except Exception as e:
    print(f"Error renaming label: {e}")
```

This snippet demonstrates a technical method, but manual renaming in the GitHub UI is probably sufficient for this specific task. 😊
